openml/datasets/dataset.py

```diff
@@ -78,10 +78,16 @@ def __init__(self, dataset_id=None, name=None, version=None, description=None,
         if features is not None:
             self.features = {}
             for idx, xmlfeature in enumerate(features['oml:feature']):
+                # split string of nominal values into type list if feature is nominal
+                # otherwise passing none for nominal values
+                try:
+                    nom_vals = [str(x[1:-1]) for x in xmlfeature['oml:nominal_values'][1:-1].split(',')]
```

Comment:
Could you please add a brief explanation of this statement, preferably with an example string that is parsed?